FirebaseCore/Sources/Private/FirebaseCoreInternal.h

```diff
@@ -23,6 +23,7 @@
 #import "FirebaseCore/Sources/Private/FIRComponentType.h"
 #import "FirebaseCore/Sources/Private/FIRDependency.h"
 #import "FirebaseCore/Sources/Private/FIRHeartbeatInfo.h"
+#import "FirebaseCore/Sources/Private/FIRHeartbeatLogger.h"
```

Comment:
Workaround for now, might be to leave it out here and only directly include it when needed by clients.

Of course, the model will not work for Swift clients which is why it may be better to go straight to FirebaseCoreInternal